docker/local.Dockerfile

```
@@ -0,0 +1,20 @@
+# Use an official Node.js runtime as a parent image
+FROM node:20
+
+# Set the working directory in the container
+WORKDIR /usr/src/app
+
+# Copy package.json and package-lock.json to the working directory
+COPY package*.json ./
+
+# Install application dependencies
+RUN npm install
```

```suggestion
Run npm ci
```
"npm i"는 실행 시, package.json에 ^ 또는 ~를 사용하는 경우 지정된 버전과 동일한 버전을 설치하지 못할 수 있다고 합니다.
따라서, 버전이 달라지면 package-lock.json이 업데이트 되므로 새로운 의존성을 설치할 때가 아니라
1. CI/CD 과정에서 전체 의존성을 설치 하거나
2. 프로젝트 레포지토리를 처음 clone 받아 의존성을 설치할 때는
"npm ci"를 사용하는 것이 좋다고 합니다.
(참고: https://doqtqu.tistory.com/353)

---

npm install -> ci로 변경했습니다

---

네 이부분은 npm install -> ci로 변경해보겠습니다